fhir-server-test/src/test/java/com/ibm/fhir/server/test/FHIRServerTestBase.java

```diff
@@ -272,6 +272,15 @@ public void beforeRequest(Map<String, List<String>> headers) {
             }
             WebSocketContainer container = ContainerProvider.getWebSocketContainer();
             Session session = container.connectToServer(endpoint, config, new URI(webSocketURL));
+            
+            // Add a Delay
+            int count = 10;
+            while ( !session.isOpen() && count > 0) {
+                System.out.println(">>> " + count + " waiting");
+                Thread.currentThread().sleep(1000l);
+                count--;
+            }
+            
```

Comment:
seems the test client always got and printed out the message correctly, so I would assume the websocket connection worked well(already opened and connected) in the failed test,  so not really sure if adding the check here can help or not...

But we definitely should try this fix!